test/test_utils/models.jl

```diff
@@ -51,3 +51,94 @@ MoGtest_default = MoGtest([1.0 1.0 4.0 4.0])
 
 # Declare empty model to make the Sampler constructor work.
 @model empty_model() = begin x = 1; end
+
+# A collection of models for which the mean-of-means for the posterior should
+# be same.
```

Comment:
I want to have a collection of models which tries out all the combinations of `*_tilde_*`, but this means that we'll sometimes have univariate latent variables rather than multivariate (e.g. `gdemo5` below). Therefore I compare the mean of the mean of the latent variables rather than the variables directly. 

Buuuuut now that we're comparing to the true mean rather than pitting the different models against each other, I guess we don't need to do that :sweat_smile: